tutorials/hana-clients-choose-hana-instance/hana-clients-choose-hana-instance.md

```diff
@@ -144,7 +144,7 @@ Congratulations! You have connected to SAP HANA Cloud and performed a few querie
 
 SAP provides a free streamlined version of SAP HANA that runs on developer laptops called [SAP HANA, express edition](https://www.sap.com/cmp/td/sap-hana-express-edition.html).
 
-SAP HANA runs on a few versions of Linux.  SAP HANA, express edition provides virtual machine images that can be run on Microsoft Windows, macOS and Linux machines as well as binary installs as described at [SAP HANA, express edition - Implement](https://help.sap.com/viewer/product/DRAFT/SAP_HANA,_EXPRESS_EDITION/latest/en-US?task=implement_task).  In terms of availability, a choice between either the server or the server + applications can be made.  The applications include the SAP HANA cockpit, the SAP HANA database explorer and the SAP HANA Web IDE.  Further details about a specific version can be found in the [release notes](https://search.sap.com/search.html?t=%22SAP%20HANA%2C%20express%20edition%202.0%20SPS%200%25%20Revision%22&n=1&s=boost&src=defaultSourceGroup).    
+SAP HANA runs on a few versions of Linux.  SAP HANA, express edition provides virtual machine images that can be run on Microsoft Windows, macOS and Linux machines as well as binary installs as described at [SAP HANA, express edition - Implement](https://help.sap.com/viewer/product/SAP_HANA,_EXPRESS_EDITION/latest/en-US?task=implement_task).  In terms of availability, a choice between either the server or the server + applications can be made.  The applications include the SAP HANA cockpit, the SAP HANA database explorer and the SAP HANA Web IDE.  Further details about a specific version can be found in the [release notes](https://search.sap.com/search.html?t=%22SAP%20HANA%2C%20express%20edition%202.0%20SPS%200%25%20Revision%22&n=1&s=boost&src=defaultSourceGroup).    
```

Comment:
Hasn't Web IDE been decommissioned?

Reply:
In late December, the recommended tool for native development for SAP HANA Cloud will switch from the SAP Web IDE to the Business Application Studio.  
The SAP Web IDE that is included in an install of SAP HANA 2.0 (on-premise and SAP HANA, express edition) will continue to be the recommended tool for native development of SAP HANA.
As the tutorial attempts to cover both SAP HANA Cloud and SAP HANA, express edition, it will continue to mention the SAP Web IDE.  I will update the tutorial once the SAP Business Application Studio supports native development for SAP HANA Cloud (late December).  At that point, there will be two tools to perform development against SAP HANA, the SAP Web IDE for on-premise and the Business Application Studio for SAP HANA Cloud.